### DISCUSSION TITLE
Migrating from version 4.0.0 to version 5.0.2

### DISCUSSION THREAD

**Discussion by bsquared938** [haugene/docker-transmission-openvpn#2670] [Q&A]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Docker run config used

(this was done on a Synology NAS so I don't have a true docker compose file, providing screenshots) 
![transmissionSettings3](https://github.com/haugene/docker-transmission-openvpn/assets/37195336/53dc03b4-b4ef-4525-9c84-69340ac9751a)
![transmissionSettings5](https://github.com/haugene/docker-transmission-openvpn/assets/37195336/223f1592-0f15-4aca-9914-b1a4e03e5bc8)

### Current Behavior

Container keeps restarting after a fatal error 

### Expected Behavior

Container should stay running 

### How have you tried to solve the problem?

1) removed pointer to VPN credentials and instead pointed to /config directly
2) removed credentials from webUI to see if that happened to be interfering 

### Log output

![image](https://github.com/haugene/docker-transmission-openvpn/assets/37195336/59f01c12-5954-4abd-ade4-55a7f93fa30e)

### HW/SW Environment

```markdown
- OS: 
- Docker:
```

### Anything else?

Things were working fine in version 4.0.0 but something has changed with config files in 5.0.2. 

I even downloaded a new OpenVPN config file from my provider and it's identical to the one that I had.  I suspect a path/volume issue is messing things up here. 

Will keep trying after I post this issue.

**Comment by pkishino** [comment]
Please read through the documentation and existing issues..this is NOT a container issue, it is a config problem on your end.
Synology are known to cause all kinds of issues here..
your logs are incomplete

**Comment by bsquared938** [comment]
Went through the documentation and a lot of other issues on here -- seems there is a lot that changed between versions.  Also the container image size is twice as big as version 4 (that alone should make it clear that a lot has changed).

I have started the container up from scratch and am slowly adding all the functionality back to it that I had before.  It is working as intended. 

For anyone who tries this: you will likely need to do the same when migrating from version 4 to 5.
